From: Ronnie Sahlberg <ronniesahlberg@gmail.com>
Date: Sat, 3 Jun 2017 11:20:20 -0700
Subject: [PATCH] Fix compiler warning/error under gcc7

Gcc7 complains about falling through from the previous case to the default arm

Signed-off-by: Ronnie Sahlberg <ronniesahlberg@gmail.com>
(cherry picked from commit 854e37aab7d7d2093a0a6620a3257b8f2a1d405c)
---
 test-tool/iscsi-support.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/test-tool/iscsi-support.c b/test-tool/iscsi-support.c
index 02367ca..ed00ce7 100644
--- a/test-tool/iscsi-support.c
+++ b/test-tool/iscsi-support.c
@@ -2930,8 +2930,6 @@ void populate_ident_tgt_desc(unsigned char *buf, struct scsi_device *dev)
                                         tgt_desig = desig;
                                         prev_type = desig->designator_type;
                                 }
-                        default:
-                                continue;
                 }
         }
         if (tgt_desig == NULL) {
